README.md

```diff
@@ -14,14 +14,30 @@ These instructions apply to minor and patch version updates. Major versions need
 a customized adaptation.
 
 After the CI is green:
-  * Update the version in the podspec
-  * Add the CocoaPods tag
-    * `git tag CocoaPods-{version}`
-    * `git push origin CocoaPods-{version}`
-  * Push the podspec to SpecsStaging
-    * `pod repo push staging GoogleDataTransport.podspec`
-  * Run Firebase CI by waiting until next nightly or adding a PR that touches `Gemfile`
-  * On google3, copybara and run a global TAP which should kick off automatically after each PR.
+* Update the version in the podspec to match the latest entry in the [CHANGELOG.md](CHANGELOG.md)
+* Checkout the `main` branch and ensure it is up to date.
+  ```console
+  git checkout main
+  git pull
+  ```
+* Add the CocoaPods tag (`{version}` will be the latest version in the [podspec](GoogleDataTransport.podspec#L3))
+  ```console
+  git tag CocoaPods-{version}
+  git push origin CocoaPods-{version}
+  ```
+* Push the podspec to SpecsStaging
+  ```console
+  pod repo push --skip-tests staging GoogleDataTransport.podspec
+  ```
+  If the command fails with `Unable to find the 'staging' repo.`, add the staging repo with:
+  ```console
+  pod repo add staging git@github.com:firebase/SpecsStaging.git
+  ```
+* Run Firebase CI by waiting until next nightly or adding a PR that touches `Gemfile`.
+* On google3, copybara and run a global TAP
+  ```console
+  third_party/firebase/ios/Releases/run_copy_bara.py --directory GoogleDataTransport --branch main
```

Comment:
The run_copy_bara.py script may not work for GDT.  Here is the last command I used for GDT:

 `/google/data/ro/teams/copybara/copybara third_party/firebase/ios/Releases/GoogleDataTransport/copy.bara.sky --piper-description-behavior=OVERWRITE --destination-cl=367099297 gdt`

So more investigation is needed here and we likely want to make gdt consistent with GoogleUtilities, but not necessary for this PR

Reply:
I'll replace the current command with this one for now...

Reply:
Figured out how to do this in g3 and updated docs accordingly in latest commit